docs/application/toc_all.md

```diff
@@ -64,7 +64,7 @@
 
 ### Natural User Interface
 #### [Overview](/application/dotnet/guides/nui/overview.md)
-#### [Setting Up the NUI Development Environment on Ubuntu](/application/dotnet/guides/nui/setup-ubuntu.md)
+#### [Quickstart](/application/dotnet/guides/nui/quickstart.md)
```

Comment:
Before SRI-B Reviewer starts review : 
According to this change, setup-ubuntu.md is replaced by quickstart.md.
However setup-ubuntu.md file itself is not removed. 
How about deleting setup-ubuntu.md file if it is not needed?

Also briefly checking this update, I got one suggestion for you.
How about merging the content of _quickstart.md_ and _hello-world.md_,
with changing the location of file to under _docs/application/dotnet/get-started_?

The described in these pages is about "how to build simple hello app",
so it looks much proper to be placed under "Get Started" section.
In that case, may the title of that page will be "Creating NUI .NET Application" or like that.

@huiyueun, Would you please let me know your opinion?

Reply:
@mijin85cho  I am starting to review the said file and hope till then @huiyueun shares the opinion.

Reply:
@mijin85cho  This app demonstrates developing a basic NUI app with custom text and animating it. 
I too think "Get started" section will be best suited for this page. 
But, I hope @huiyueun opinion is most valuable in this regard,

Reply:
1. I will remove setup-ubuntu.md file
2. I think it's good to be moved to "Get started" page.
I will modify it for the 'Get Started' form and merge it with the 'hello-world' page.
and I will re-upload the patch fixed.
Thank you.

Reply:
@huiyueun 
Thanks for understanding.